src/plugins/expressions/public/index.ts

```diff
@@ -30,7 +30,7 @@ export function plugin(initializerContext: PluginInitializerContext) {
 }
 
 // Static exports.
-export { ExpressionExecutor, IExpressionLoaderParams } from './types';
+export { ExpressionExecutor, IExpressionLoaderParams, RenderError } from './types';
```

Comment:
Since we've been trying to prepend type names with namespaces of the services they belong to, maybe we should be exporting this as `ExpressionRenderError` now that it is added to the public contract? `RenderError` feels very generic.

Reply:
You just couldn't let this PR contain just 5 files :-}

Reply:
🙈